### ISSUE TITLE
  Hello, I was discussing with you Guys...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#4040]
<p>&nbsp;&nbsp;Hello,<br/>I was discussing with you Guys at the Worm meeting the aspect of finding the list of genes that have dpl-1 binding site/region. Is it possible to get such a lit of genes, or at least a number of genes (not number of binding sites, as there can be few of them per gene). That would be so very useful for finishing up my manuscript. Thank you for your time - Karolina</p><p>What were you doing? <br />&nbsp;&nbsp;postdoc at MDI Biological Laboratory</p>

Reported by: Karo*********************** (kand****************)  (obscured for privacy))
Submitted From: support <a href="http://www.wormbase.org//tools/support">/tools/support</a>

Mozilla/5.0 (Windows NT 6.3; WOW64; Trident/7.0; Touch; .NET4.0E; .NET4.0C; .NET CLR 3.5.30729; .NET CLR 2.0.50727; .NET CLR 3.0.30729; Tablet PC 2.0; rv:11.0) like Gecko

**Comment by xdwang** [comment]
Hi Karolina,

I did some research for you.  In wormbase, we have 11248 sequence features including dpl-1 binding sites/region. They are all from Julie Ahringer lab’s modEncode data. 
It’s published in this paper:

http://genome.cshlp.org/content/24/7/1138.long <http://genome.cshlp.org/content/24/7/1138.long>

You may need some helps to get the list of genes. 
Let’s see what’s the suggestion from our sequence feature curators.

Best, 

Xiaodong



> On Jul 10, 2015, at 9:11 AM, no-reply@wormbase.org wrote:
> 
> How Do I ...? 
>   Hello,
> I was discussing with you Guys at the Worm meeting the aspect of finding the list of genes that have dpl-1 binding site/region. Is it possible to get such a lit of genes, or at least a number of genes (not number of binding sites, as there can be few of them per gene). That would be so very useful for finishing up my manuscript. Thank you for your time - Karolina
> 
> What were you doing? 
>   postdoc at MDI Biological Laboratory
> 
> Submitted From: 
>   support /tools/support <http://www.wormbase.org//tools/support>
> This issue has been assigned ticket number: #4040. Follow progress on our issue tracker <https://github.com/WormBase/website/issues/4040>.
> 
> 
> Reported by: Karolina de Groot-Andralojc (kandralojc@mdibl.org <mailto:kandralojc@mdibl.org>) 
> 10 Jul 15 13:11 
> 
> Mozilla/5.0 (Windows NT 6.3; WOW64; Trident/7.0; Touch; .NET4.0E; .NET4.0C; .NET CLR 3.5.30729; .NET CLR 2.0.50727; .NET CLR 3.0.30729; Tablet PC 2.0; rv:11.0) like Gecko 
> 
> This question was reported via the web interface at wormbase.org <http://wormbase.org/>. You can also email us <mailto:help@wormbase.org> directly.
> 
> 
> -- 
> -- 
> You received this message because you are subscribed to the "WormBase: help" group.
> To post to this group, send email to help@wormbase.org <mailto:help@wormbase.org>
> To unsubscribe from this group, send email to
> help+unsubscribe@wormbase.org <mailto:help+unsubscribe@wormbase.org>
> For more options, visit this group at
> http://groups.google.com/a/wormbase.org/group/help?hl=en <http://groups.google.com/a/wormbase.org/group/help?hl=en>
> 
> 
> To unsubscribe from this group and stop receiving emails from it, send an email to help+unsubscribe@wormbase.org <mailto:help+unsubscribe@wormbase.org>.

Xiaodong Wang, Ph.D
Wormbase Curator
626-395-8079
Biology Division
California Institute of Technology
1200 E. California Blvd
Pasadena, CA 91125

**Comment by chris-grove** [comment]
@gw3  @khowe @Paul-Davis @epaule 
Do we have any existing (or "easily" modifiable/evolvable) pipelines or scripts that could generate a list of "target" genes given a DNA binding protein with data from genome-wide (ChIP-chip, ChIP-Seq) chromatin immunoprecipitation studies? I spoke with Karolina at the meeting about this and have been meaning to get back to her about it. That could be a very valuable tool for any DNA binding protein.

**Comment by chris-grove** [comment]
To follow up, I think we'd theoretically need to specify at least three things:
1) what the boundaries of any "gene" are; we could focus on protein coding
2) generic ChIP signal cutoff to indicate a binding event
3) number of sites required to consider a "target" gene
I imagine that 1 & 2 may be quite tricky sometimes

**Comment by gw3** [comment]
I don't really have any expertise in ChIP-seq analysis.
The best pipeline I can find is Galaxy:

https://sites.google.com/site/princetonhtseq/tutorials/chip-seq

**Comment by chris-grove** [comment]
@gw3 OK thanks. Looks like we'd want the "MACS" software option to detect likely transcription factor binding sites. Is this something we could adapt and implement?

**Comment by gw3** [comment]
It depends what their experimental design is like.
MACS is certainly one peak-caller for narrow peaks, but there is also: DiffBind, SPP, PeakSplitter and several Bioconductor packages.
I really have no expertise in this, I just spent a few minutes looking through SeqAnswers.

**Comment by khowe** [comment]
@chris-grove @gw3 Wait a minute. Is the suggestion that we re-analyse of the modENCODE ChIP data, and re-call peaks (i.e. binding sites) for each transcription factor, using whatever tool is flavour of the month (be it MACS, SwEMBL, or whatever)? If that's being suggested, then I'm afraid that's beyond our resources at this time. 

However, the modENCODE projects each did their own analyses, called their own peaks, and as @xdwang says above, we have created sequence features on the genome, linked to objects (in ?Transcript_factor class) that denote which factor is being bound. So, the query of interest would be:

1. Find the ?ranscription_factor object corresponding to the gene of interest (in this case, dpl-1)
2. Find all sequence features that are linked to this ?Transcription_factor
3. Find the genes "associated" with those binding features. 

(3) remains an messy problem as far as I am aware, because many/most binding events will be outside of the extent of genes. So then you have to start using distance cut-offs (e.g. within 500 bps). 

But to me, this is very well suited to a WormMine workflow (or at least it is the kind of workflow that WormMine should be supporting). Logically, it's two steps: (a) get me a list of genomic regions having a particular property; and (2) give me features (e.g. genes) that lie within a specified distance of the regions found in (a).

**Comment by chris-grove** [comment]
@khowe  Yes, it would be nice to have such a query in WormMine, although I think that will likely be a ways off. The only sequence features in WormMine so far, as far as I can tell, are CDS, allele, chromosome, gene, and transcript. But I agree that WormMine in general is well suited to this task. I'll talk to Todd and the WormMine team about incorporating other sequence features including transcription factor binding sites.

**Comment by chris-grove** [comment]
@khowe @gw3  
Todd has the loading of sequence features into WormMine on his to do list. The remaining question is whether or not we can do the query we want with existing WormMine functionality, as we don't have control over development of Intermine

**Comment by tharris** [comment]
Which is to say (3).
